pages/linux/ddcutil.md

```diff
@@ -0,0 +1,20 @@
+# ddcutil
+
+> Control the settings of connected displays via DDC/CI.
+> More information: <https://www.ddcutil.com>.
+
+- List all compatible displays:
+
+`ddcutil detect`
+
+- Change the brightness (option 0x10) of display 1 to 50%:
+
+`ddcutil -d {{1}} setvcp {{10}} {{50}}`
```

Comment:
We use long option when possible. Please also change this in the other examples:
```suggestion
`ddcutil --display {{1}} setvcp {{10}} {{50}}`
```